staging/src/k8s.io/apiserver/pkg/endpoints/filters/authorization.go

```diff
@@ -70,7 +70,7 @@ func WithAuthorization(handler http.Handler, a authorizer.Authorizer, s runtime.
 			return
 		}
 
-		klog.V(4).Infof("Forbidden: %#v, Reason: %q", req.RequestURI, reason)
+		klog.V(4).InfoS("Forbidden", "URI", req.RequestURI, "Reason", reason)
```

Comment:
mind showing a before and after for this line in the log?

Reply:
before:
I0120 10:22:09.051284    1668 authorization.go:73] Forbidden: "", Reason: "RBAC: not allowed to patch pod"
after:
I0120 10:22:09.051284    1668 authorization.go:74] "Forbidden" URI="" Reason="RBAC: not allowed to patch pod"